components/layout/app-bar.jsx

```diff
@@ -25,8 +26,14 @@ const DashboardAppBar = ({ children, className, ...restProps }) => {
       </AppBar.Brand>
 
       {children}
+      <AppBar.Item>
+        {`${store.user.givenName} ${store.user.familyName} `}|{' '}
+        <Button className={styles.logoutButton} onClick={store.logout}>
+          Logout
+        </Button>
+      </AppBar.Item>
```

Comment:
Can you write this as a separate component that will be connected to store but the AppBar will not?

Reply:
sure